ctms/sync.py

```diff
@@ -90,12 +92,15 @@ def sync_records(self, db, end_time=None):
         all_acoustic_records_before_now: List[
             PendingAcousticRecord
         ] = get_all_acoustic_records_before(
-            db, end_time=end_time, retry_limit=self.retry_limit
+            db,
+            end_time=end_time,
+            retry_limit=self.retry_limit,
+            batch_limit=self.batch_limit,
         )
 
         # For each record, attempt downstream sync
         for acoustic_record in all_acoustic_records_before_now:
             self._sync_pending_record(db, acoustic_record)
-        # Commit changes to db after all records are processed
-        db.commit()
+            # Commit changes to db after EACH record is processed
```

Comment:
ironically, if we're only doing 20 at a time, this probably  isn't necessary :) 🤷

Reply:
Reverted